### PR TITLE
Add support for type Bearer authorization header

### PR DESCRIPTION
**Pull request by wmichelin**

### CODE REVIEW

**Review by yuehlin — APPROVED** [review]

**Review by dvdpearson — APPROVED** [review]

File: lib/Shippo/ApiRequestor.php [hunk 1]
```diff
@@ -155,7 +155,11 @@ private function _interpretResponse($rbody, $rcode)
         }
         return $resp;
     }
-    
+
+    private function _getAuthorizationType($apiKey = '')
```
Comment (ketzusaka) [inline lib/Shippo/ApiRequestor.php:159]: Is there a case where you'd want this to be an empty string?

File: lib/Shippo/ApiRequestor.php [hunk 2]
```diff
@@ -155,7 +155,11 @@ private function _interpretResponse($rbody, $rcode)
...
+    {
+        return strpos($apiKey, 'oauth.') === 0 ? 'Bearer' : 'ShippoToken';
```
Comment (ketzusaka) [inline lib/Shippo/ApiRequestor.php:161]: Is this line tested somewhere?

Comment (wmichelin) [reply lib/Shippo/ApiRequestor.php]: No

File: lib/Shippo/ApiRequestor.php (hunk 1 above)
Comment (wmichelin) [reply lib/Shippo/ApiRequestor.php:159]: No, but I'd like to have it default to string

File: lib/Shippo/ApiRequestor.php (hunk 2 above)
Comment (wmichelin) [reply lib/Shippo/ApiRequestor.php:161]: added unit test in https://github.com/Weebly/shippo-php-client/pull/1/commits/ab2d5368f96c0734dde56b9ea5403fdea4692753

File: lib/Shippo/ApiRequestor.php (hunk 1 above)
Comment (ketzusaka) [reply lib/Shippo/ApiRequestor.php:159]: Line 116 there's a check to see `if (!$myApiKey)` and it'll throw an error. I don't know PHP super well, but my gut tells me that'll pass on an empty string and it should probably throw the same error if its an empty string.

Comment (ketzusaka) [reply lib/Shippo/ApiRequestor.php:159]: Nevermind, PHP does handle this as it should be!

**Review by ketzusaka — APPROVED** [review]

**Pull request merged**